lib/ansible/modules/clustering/kubevirt/kubevirt_cdi_upload.py

```diff
@@ -0,0 +1,205 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2018, KubeVirt Team <@kubevirt>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+
+module: kubevirt_cdi_upload
+
+short_description: Upload local VM images to CDI Upload Proxy.
```

Comment:
I Would add some note/link or something how to deploy CDI

Reply:
I believe CDI is included as standard in CNV and in kubevirt's dev env? @pkliczewski 

I did add a link to exposing the upload proxy itself in the description of `upload_host` parameter though.